azure-pipelines.yml

```diff
@@ -159,11 +153,26 @@ stages:
               publisherId: "$(publisher)"
               extensionId: "sonarcloud"
               accounts: "https://devops.azure.com/$(itOrg)"
+          - task: Bash@3
+            displayName: "Wait for tasks to be available in CI"
+            env:
+              SERVICE_URL: "https://devops.azure.com/$(AzdoItsOrga)"
+              AZURE_TOKEN: $(ItOrgPat)
```

Comment:
I created a new PAT `ItOrgPat` on AzDO variable group to be able to run `tfx` in the test org

Reply:
seems I need this change in my PR too as the race condition is present on master, will wait for this to be merged to rebase